gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-version=1.0.0
+version=1.1.0
```

Comment:
If we're following semantic versioning, it is hard to say what version this change should be.  On one hand, you could argue this change is backwards compatible and should be a 1.0.1 release.  On the other hand, if anyone is using the library and giving it bad path configuration, expecting it to work the same, you could argue that this change is definitely not backwards compatible and should be a 2.0.0 release.  Any preference?